src/app.js

```diff
@@ -383,6 +384,25 @@ function run () {
   })
   var offsetToLineColumnConverter = new OffsetToLineColumnConverter(compiler.event)
 
+  // ---------------- ContextualListener -----------------------
+  this._components.contextualListener = new ContextualListener({
+    getCursorPosition: () => {
+      return this._components.editor.getCursorPosition()
+    },
+    getCompilationResult: () => {
+      return compiler.lastCompilationResult
+    },
+    warnExpression: (position) => {
+      position = offsetToLineColumnConverter.offsetToLineColumn(position, position.file, compiler.lastCompilationResult)
+      return editor.addMarker(position, config.get('currentFile'), 'highlightcall')
```

Comment:
Be careful - references across files are easily possible.

Reply:
Yes now only one sourceunit is managed. but that stuff need to be fixed when we'll add other feature...

Reply:
Sure, that's fine, I just did not see the check makes sure that `position` and `config.get('currentFile')` reference the same file.